### DISCUSSION TITLE
When will this be released to the chrome webstore?

### DISCUSSION THREAD

**Discussion by Explosion-Scratch** [ScratchAddons/ScratchAddons#264] [Help]
Title.

**Comment by WorldLanguages** [comment]
¯\\_(ツ)_/¯

**Comment by Explosion-Scratch** [comment]
> ¯\_(ツ)_/¯

lol
![image](https://user-images.githubusercontent.com/61319150/92960118-a6ab2e80-f432-11ea-9220-ff89cf0196da.png)

**Comment by jeffalo** [comment]
> ¯\_(ツ)_/¯

\ you dropped this

**Comment by GrahamSH-LLK** [comment]
It will be released this week!
